src/sql/azdata.proposed.d.ts

```diff
@@ -1802,9 +1802,9 @@ declare module 'azdata' {
 
 	export interface TextComponentProperties {
 		/**
-		 * Corresponds to the aria-live accessibility attribute for this component
+		 * Corresponds to the aria-live accessibility attribute for this component. The accepted values are polite, assertive, and off
 		 */
-		ariaLive?: string;
+		ariaLive?: 'polite' | 'assertive' | 'off';
```

Comment:
Please define an actual new type and use that

`export type AriaLiveValue = 'polite' | 'assertive' | 'off';`

Reply:
And then also please update the InputBoxProperties to use this new type

`ariaLive?: AriaLiveValue | undefined;`

It's ok to update azdata.d.ts in this case, it's not a runtime breaking change and if people were passing other values it wouldn't have worked anyways so it's fine to introduce a compile error in this case.